src/main/webapp/script.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+* Add a href link to the logout button if the user is logged in,
```

Comment:
This function should handle user authentication only. Generally speaking, we should extract all other functionalities to new functions. 
In this case, extract only the part which handles the logoutButton, since the other part consists of one line only.

Reply:
done. i created a method called `addLinktoLogoutButton` to handle it.